java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -155,7 +163,45 @@ void executeInstruction(BytecodeCFGBuilder.Instruction instruction) {
   }
 
   private void handleBlockExit(ProgramPoint programPosition) {
-    programPosition.block.successors().forEach(b -> enqueue(new ProgramPoint(b), programState));
+    BytecodeCFGBuilder.Block block = (BytecodeCFGBuilder.Block) programPosition.block;
+    BytecodeCFGBuilder.Instruction terminator = block.terminator();
+    if (terminator != null) {
+      switch (terminator.opcode) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)